extensions/amp-connatix-player/0.1/test/validator-amp-connatix-player.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
```

Comment:
Please include the license at the top of the file:

```
<!--
  Copyright 2019 The AMP HTML Authors. All Rights Reserved.
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
      http://www.apache.org/licenses/LICENSE-2.0
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS-IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the license.
-->
<!--
  Test Description:
  Tests for the amp-connatix-player tag.
-->
```

Reply:
fixed